bindings/pyfiction/include/pyfiction/inout/write_svg_layout.hpp

```diff
@@ -17,25 +17,79 @@
+            return oss.str();                                     // Return the string content
+        },
+        "layout"_a, "ps"_a = fiction::write_sidb_layout_svg_params{}, DOC(fiction_write_sidb_layout_svg));
+
+    // QCA plot
+    void (*write_qca_layout_svg_pointer)(const py_qca_layout&, const std::string_view&,
+                                         const fiction::write_qca_layout_svg_params&) =
+        &fiction::write_qca_layout_svg<py_qca_layout>;
+
+    m.def("write_qca_layout_svg", write_qca_layout_svg_pointer, "layout"_a, "filename"_a,
+          "params"_a = fiction::write_qca_layout_svg_params{}, DOC(fiction_write_qca_layout_svg));
+}
+
+}  // namespace detail
+
+void write_svg_layout(pybind11::module& m)
 {
     namespace py = pybind11;
     using namespace pybind11::literals;
 
+    py::enum_<fiction::color_mode>(m, "color_mode", DOC(fiction_color_mode))
+        .value("LIGHT", fiction::color_mode::LIGHT, DOC(fiction_color_mode_LIGHT))
+        .value("DARK", fiction::color_mode::DARK, DOC(fiction_color_mode_DARK));
+
+    py::class_<fiction::write_sidb_layout_svg_params>(m, "write_sidb_layout_svg_params",
+                                                      DOC(fiction_write_sidb_layout_svg_params))
+        .def(py::init<>())
+        .def_readwrite("lattice_point_size", &fiction::write_sidb_layout_svg_params::lattice_point_size,
+                       DOC(fiction_write_sidb_layout_svg_params_lattice_point_size))
+        .def_readwrite("sidb_size", &fiction::write_sidb_layout_svg_params::sidb_size,
+                       DOC(fiction_write_sidb_layout_svg_params_sidb_size))
+        .def_readwrite("sidb_border_width", &fiction::write_sidb_layout_svg_params::sidb_border_width,
+                       DOC(fiction_write_sidb_layout_svg_params_sidb_border_width))
+        .def_readwrite("color_background", &fiction::write_sidb_layout_svg_params::color_background,
+                       DOC(fiction_write_sidb_layout_svg_params_color_background));
+
     py::class_<fiction::write_qca_layout_svg_params>(m, "write_qca_layout_svg_params",
                                                      DOC(fiction_write_qca_layout_svg_params))
         .def(py::init<>())
         .def_readwrite("simple", &fiction::write_qca_layout_svg_params::simple,
-                       DOC(fiction_write_qca_layout_svg_params_simple))
-
-        ;
+                       DOC(fiction_write_qca_layout_svg_params_simple));
+    ;
 
-    void (*write_svg_layout_function_pointer)(const py_qca_layout&, const std::string_view&,
-                                              const fiction::write_qca_layout_svg_params&) =
-        &fiction::write_qca_layout_svg<py_qca_layout>;
+    detail::write_svg_layout_impl<py_charge_distribution_surface_100>(m);
+    detail::write_svg_layout_impl<py_charge_distribution_surface_100>(m);
+    detail::write_svg_layout_impl<py_charge_distribution_surface>(m);
```

Comment:
```suggestion
    detail::write_svg_layout_impl<py_charge_distribution_surface_111>(m);
    detail::write_svg_layout_impl<py_charge_distribution_surface_100>(m);
    detail::write_svg_layout_impl<py_charge_distribution_surface>(m);
```

Still the same copy-paste-error.